examples/worlds/linear_battery_demo.sdf

```diff
@@ -28,7 +28,7 @@
   Stop recharging the battery using a topic (optional if <recharge_by_topic> is true):
       ign topic -t /model/vehicle_blue/battery/linear_battery/recharge/stop -m ignition.msgs.Boolean -p 'data:true'
 
-  The blue vehicle should stop when it runs out of battery.
+  After the battery runs out, no force is applied to the joints in the blue vehicle. It would keep on moving as there is no friction to stop it.
```

Comment:
Alternatively, we can joint friction to the vehicles which would bring them to a stop. I still think what you added here is helpful. We can say:
>   After the battery runs out, no force is applied to the joints in the blue vehicle. It eventually comes to a stop due to friction in its joints. Had there been no joint friction, the vehicle would have kept on moving as Gazebo currently does not support rolling friction.

Reply:
Added here : https://github.com/gazebosim/gz-sim/pull/1500/commits/cce79a3a64c6deb53bde1e29c351438516b64da3